api/libs/scenarios-planning-unit/src/scenarios-pu-pa-data.geo.entity.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  OneToOne,
+  PrimaryGeneratedColumn,
+  RelationId,
+} from 'typeorm';
+import { ScenariosPlanningUnitGeoEntity, ScenariosPuCostDataGeo } from '@marxan/scenarios-planning-unit';
+import { PlanningUnitsGeom } from '@marxan-jobs/planning-unit-geometry/planning-units.geo.entity';
+
+@Entity({
+  name: 'scenarios_pu_data',
+})
+export class ScenariosPuPaDataGeo extends ScenariosPlanningUnitGeoEntity {
+
+  @Column({
+    type: 'float8',
+    nullable: true,
+    name: 'protected_area',
+  })
+  protectedArea!: number;
+
+  @OneToOne(() => PlanningUnitsGeom, PlanningUnitsGeom => PlanningUnitsGeom.id, {
+    onDelete: 'CASCADE',
+  })
+  @JoinColumn({
+    name: 'pu_geom_id',
+    referencedColumnName: 'id',
+  })
+  planning?: PlanningUnitsGeom | null;
```

Comment:
I suggest to make this property name more descriptive - this is referencing a full planning unit geometry, right? if so, what about `planningUnitGeom`?

Reply:
I would also suggest to add the cascades in the actual table definitions via a migration (I think they're not there)